package/package.json

```diff
@@ -1,20 +1,17 @@
 {
   "name": "react-native-winrt",
-  "version": "0.0.1",
+  "version": "0.67.0",
   "license": "MIT",
   "description": "React Native WinRT is a module for React Native for Windows developers who want to call native (non-XAML) WinRT APIs in their apps",
   "main": "index.js",
+  "types": "index.d.ts",
   "repository": {
     "type": "git",
     "url": "git@github.com/microsoft/react-native-winrt.git"
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "files": [
-    "LICENSE",
-    "README.md"
-  ],
```

Comment:
The files that are included are now controlled by the `.npmignore` file (exclusion instead of inclusion) since that just became simpler and easier: https://github.com/microsoft/react-native-winrt/blob/dunhor/autolinking/package/.npmignore